client/dashboard/style.scss

```diff
@@ -0,0 +1,22 @@
+/** @format */
+
+.woocommerce-dashboard__columns {
```

Comment:
You can make this file much simpler & not rely on negative margins by using css grid:

```scss
.woocommerce-dashboard__columns {
	display: grid;
	grid-template-columns: 1fr 1fr;
	grid-column-gap: $gutter;

	@include breakpoint( '<782px' ) {
		grid-template-columns: 1fr;
	}
}
```

I would also avoid using px values for any spacing, we should always use our variables - unless it's a CSS magic number situation, where it needs to be absolutely positioned or something, but then it should be commented.

Reply:
TIL thank you.